source/service/k3s.rst

```diff
@@ -0,0 +1,952 @@
+.. _service_k3s:
+
+============================
+Lightweight Kubernetes (K3s)
+============================
+
+OpenNebula `Marketplace Appliance <https://marketplace.opennebula.io/appliance/319d21ea-1a39-4c28-bc36-5ac86a966a48>`_  with preinstalled `K3s <https://k3s.io/>`_ service.
+
+Without any parameters provided, the appliance deploys as a single Kubernetes master node (K3s flavour). Appliance can be customized with :ref:`contextualization <k3s_context_param>` parameters to support manually managed multi-node deployments, or automatically managed multi-node cluster as OpenNebula `OneFlow service <https://docs.opennebula.io/stable/management_and_operations/multivm_service_management/appflow_use_cli.html>`_.
+
+.. include:: shared/features-alpine.txt
+* **Only single master clusters supported**.
+* Supports multiple CNI networking: `Flannel <https://github.com/coreos/flannel>`_, `Calico <https://www.projectcalico.org/>`_ and Canal (Calico+Flannel).
+* `MetalLB <https://metallb.universe.tf/>`_ LoadBalancer is supported as an alternative to the default K3s servicelb.
+
```

Comment:
I would add here as information for the user the default technologies for CSI and ingress controller bundled with k3s distribution i.e. local path provisioner (https://github.com/rancher/local-path-provisioner) and traefik (https://traefik.io/traefik)